### ISSUE TITLE
no-jquery does not catch $.extend() in a service

### ISSUE THREAD

**Issue by tomwayson** [ember-cli/eslint-plugin-ember#279]
I just [configured eslint](https://github.com/Esri/ember-arcgis-portal-services/pull/135/files#diff-e4403a877d80de653400d88d85e4801a) to use `"ember/no-jquery": 2,` and it's not picking up on [a service in my addon](https://github.com/Esri/ember-arcgis-portal-services/pull/135/files#diff-5c43a7a0cb851787474c7cc039be1abf) that uses `import $ from 'jquery';` and then `let ops = $.extend({}, defaults, options);`.

I would have expected that to have been caught. I wonder if I've configured it wrong somehow? I'm using this same rule in other addons/apps. Here's my env:

```
ember-cli: 2.18.2
node: 8.9.4
os: darwin x64
```

**Comment by bmish** [comment]
Fixed by #1063.
